src/SyncDriver.cpp

```diff
@@ -45,9 +45,6 @@ void SyncDriver::startMove(long steps1, long steps2, long steps3){
      * Initialize state for all active motors
      */
     FOREACH_MOTOR(
-        if (steps[i]){
-            motors[i]->startMove(steps[i]);
-        };
```

Comment:
Thank you for your research into this problem and finding the cause.

Removing this block unfortunately results in two more issues
1. startMove() not being called at all in CONSTANT_SPEED mode;
2. the synced timings are not recalculated any more (`startMove()` is needed to recalculate the timings after the rpm was changed)